odl/space/numpy_ntuples.py

```diff
@@ -563,7 +587,7 @@ class Fn(FnBase, Ntuples):
     Its elements are represented as instances of the `FnVector` class.
     """
 
-    def __init__(self, size, dtype, **kwargs):
+    def __init__(self, size, dtype='float64', **kwargs):
```

Comment:
Well yes, but this is the same as the value of `default_type()`, which seemed reasonable?